### PR TITLE
Add the thousand validators page

### LINKED ISSUE #1383: Add Thousand Validators Page
# Kusama:

## Requirements

- Verified Identity
- Connect to dedicated telemetry
- Min 50 KSM Self Bonded
- no more than 10% commission 
- separate controller and stash
- must be on latest release
- Max two nodes (under same sub/super identity)

## Applying & Setup their node
 

---

# Polkadot

- Connect to Dedicated Telemetry
- Verified Identity
- Rank 25 or higher on Kusama Thousand Validators Programme
- Min 10K DOTs self staked (approval for good intentions)
- Reward Destination 'Staked'
- No more than 3% commission
- Separate Stash and Controller
- Must be on latest release

## Applying  & Setup their node

### PR DESCRIPTION
**Pull request by lsaether**
closes #1383

### CODE REVIEW

File: docs/thousand-validators.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## How to Apply
+
+### Kusama
+
+In order to apply to the Kusama programme, set up your node to adhere to the requirements below and
+fill in the [application form][kusama 1kv form]. You will hear back from the team shortly.
+
+#### Requirements
+
+- Verified identity
```
Comment (ansonla3) [inline docs/thousand-validators.md]: add link https://guide.kusama.network/docs/en/mirror-learn-identity#setting-an-identity

File: docs/thousand-validators.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Connect to dedicated telemetry
```
Comment (ansonla3) [inline docs/thousand-validators.md]: add --telemetry-url 'wss://telemetry-backend.w3f.community/submit 1'

File: docs/thousand-validators.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- Minimum of 50 KSM self-stake
+- No more than 10% commission
+- Separate controller and stash (or have a Staking proxy set up)
+- Must be on latest release
+- Max two nodes (under same sub/super identity)
+
+### Polkadot
+
+> Note: Entrance to the Polkadot programme requires a rank of 25 or higher in the Kusama programme.
```
Comment (ansonla3) [inline docs/thousand-validators.md:54]: add https://thousand-validators.kusama.network/#/leaderboard for checking

File: docs/thousand-validators.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- Rank 25 or higher on Kusama Thousand Validators Programme
+- Mininmum of 10K DOTs self stake (exceptions by approval for good intentions)
+- Reward destination 'Staked'
+- No more than 3% commission
+- Separate stash and controller (or have a Staking proxy set up)
+- Must be on latest release
+
+[kusama 1kv form]: https://forms.gle/xqYLoceTwg1qvc9i6
+[polkadot 1kv form]:
+  https://docs.google.com/forms/d/e/1FAIpQLSdS-alI-J2wgIRCQVjQC7ZbFiTnf36hYBdmO-1ARMjKbC7H9w/viewform
```
Comment (ansonla3) [inline docs/thousand-validators.md:74]: would be great to add the 1kv channel link here for asking questions

Comment (ansonla3) [reply docs/thousand-validators.md:74]: and guide on how to set up the validator as well

File: docs/thousand-validators.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+It serves two major purposes: 1) to give validators a structured on-ramp to join the active set of
+validators on Kusama and Polkadot and 2) to further decentralize the validator active set.
+
+## How it Works
+
+The nominating backend will routinely change its nominations at every era (or every 4 eras on
+Kusama). It will nominate validators which fit all the requirements and are eligbile. Of this pool,
+it will nominate as many as possible although some validators which are eligible might not receive
+nominations every round (due to the constraint of nominating a maximum of 16 validators per
+nominator). If a validator performs well during a single nomination period (the time after a new
```
Comment (ansonla3) [inline docs/thousand-validators.md]: hmm..interesting.. my node keeps dropping for the past few days..;). could you explain a little bit more on "performs well"?

**Comment by laboon** [comment]
@lsaether Can you also add a link to this page from the run a validator page?  https://wiki.polkadot.network/docs/en/maintain-guides-how-to-validate-polkadot

**Comment by laboon** [comment]
It looks like Logan has addressed all of Anson's and my feedback, so I am merging.

**Pull request merged**